hardhat.config.js

```diff
@@ -50,8 +50,11 @@ module.exports = {
         hardhat: {
             chainId: 31337,
             forking: {
-                url: MAINNET_RPC_URL,
+                url: MUMBAI_RPC_URL,
```

Comment:
I have changed to mumbai rpc and the tests didn't work. Could you confirm that on your machine works ?